rust/stub/Cargo.toml

```diff
@@ -7,12 +7,11 @@ publish = false
 rust-version = "1.68"
 
 [dependencies]
-uefi = { version = "0.20.0", default-features = false, features = [ "alloc", "global_allocator" ] }
-uefi-services = { version = "0.17.0", default-features = false, features = [ "panic_handler", "logger" ] }
+uefi = { version = "0.21.0", default-features = false, features = [ "alloc", "global_allocator" ] }
+uefi-services = { version = "0.18.0", default-features = false, features = [ "panic_handler", "logger" ] }
 goblin = { version = "0.6.1", default-features = false, features = [ "pe64", "alloc" ]}
 bitflags = "2.2.1"
 
-# Even in debug builds, we don't enable the debug logs, because they generate a lot of spam from goblin.
```

Comment:
Was this deleted by mistake? Doesn't really matter much.

Reply:
Yes, will restore it